docs/how-to/configure/synchronous-replication.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Synchronous Replication
+summary: Learn how to configure synchronous replication in dual data centers.
+category: how-to
+---
+
+# Synchronous Replication
+
+This section describes how to configure synchronous replication in dual data centers.
+
+One of the data centers is primary, and the other is dr. When a region has odd number of replicas, primary will have more. When dr is down for more than configured time, replication will be changed to be asynchronous and primary will serve requests by its own. Asynchronous replication is used by default.
```

Comment:
Maybe we should describe more detail about these two replication modes (like how a raft log is considered committed).

Reply:
It's a configuration documents, just talk how to enable the feature. A details introduction is expected to be a reference.